### PR TITLE
frontend apis call

### PR DESCRIPTION
**Pull request by JazibEqbal**

### CODE REVIEW

File: src/lib/api.js [hunk 1]
```diff
@@ -36,6 +36,75 @@ class YufaabInstance {
...
+    try {
+      const options = {
+        method: 'POST',
+        url: `${this.host}/order`,
+        headers: {
+          Authorization: `JWT ${this.getToken()}`
+        },
+        data: {
+          data,
+          status,
```
Comment (imfAli) [inline src/lib/api.js:78]: Whats this status?

File: src/lib/api.js [hunk 2]
```diff
@@ -36,6 +36,75 @@ class YufaabInstance {
...
+
+  async updateOrder(data) {
+    try {
+      const options = {
+        method: 'POST',
+        url: `${this.host}/order`,
+        headers: {
+          Authorization: `JWT ${this.getToken()}`
+        },
+        data: {
```
Comment (imfAli) [inline src/lib/api.js:97]: We need to pass orderid in case of update

Comment (JazibEqbal) [reply src/lib/api.js:96]: see now

Comment (imf-ali) [reply src/lib/api.js:96]: No bro, order id is being passed in body, refer to the curl that was sent

**Pull request merged**